components/page-aside/PageAside.vue

```diff
@@ -53,6 +60,14 @@ export default {
       }
       return src
     },
+    partnerSrc: function () {
+      var src = _defaultAvatarSrc
+      const partner = this.partner.logo
+      if (partner && partner.imgixHost) {
```

Comment:
This feels like a duplicate, why not put the src immediately in src of the image?